owsi-core/owsi-core-components/owsi-core-component-commons/src/main/java/fr/openwide/core/commons/util/binding/BindingUtils.java

```diff
@@ -0,0 +1,20 @@
+package fr.openwide.core.commons.util.binding;
+
+import org.bindgen.Binding;
+
+public final class BindingUtils {
+
+	public static Class<?> getRootType(Binding<?> binding) {
```

Comment:
If we do this, we need to switch from using Binding<?> in the API to using AbstractCoreBinding<?>. Is it something we want?